src/node.h

```diff
@@ -484,6 +484,8 @@ extern "C" NODE_EXTERN void node_module_register(void* mod);
  */
 NODE_EXTERN void AtExit(void (*cb)(void* arg), void* arg = 0);
 
+extern bool config_preserve_symlinks;
```

Comment:
That was the first place I looked and I could swear it wasn't there this morning... at any rate, I don't think node.h is the right place to declare it; maybe node_internals.h.